make/e2e-targets.mk

```diff
@@ -62,7 +68,7 @@ endif
 e2e-test: kill-httpserver build-mock start-httpserver create-config do-e2e-test
 
 remote-e2e-test: build-mock do-remote-e2e-test
-do-remote-e2e-test: create-config
+do-remote-e2e-test: clean-regressions create-config
```

Comment:
At first I was just clearing `.regression.png` but then I thought, "those darn `.png` files bug me too, so I removed them as well, but forgot to update the target dependency there. Thanks.